pages/k8s/cni-multus.md

```diff
@@ -0,0 +1,224 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "CNI with Multus"
+  description: How to deploy and use Multus in Charmed Kubernetes
+keywords: CNI, networking, multus
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: cni-multus.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+[Multus][multus] is a CNI plugin for Kubernetes that enables attaching multiple
+network interfaces to pods. Multus support for **Charmed Kubernetes** is
+provided by the Multus charm, which must be deployed into a Kubernetes model
+in Juju.
+
+## Requirements
```

Comment:
It would be worth calling out that you still need to have at least one of the current CNI subordinates attached to your cluster as a prerequisite.

Reply:
done